packages/gatsby-theme-blog/src/components/seo.js

```diff
@@ -10,7 +10,7 @@ import PropTypes from "prop-types"
 import Helmet from "react-helmet"
 import { useStaticQuery, graphql } from "gatsby"
 
-function SEO({ description, lang, meta, keywords, title }) {
+function SEO({ description, lang, meta, keywords, title, imageSource }) {
```

Comment:
Should this be added to `SEO.propTypes` below?

Reply:
It should indeed. At the moment this branch is based on existing open PRs. So I've gone ahead and made this change in #23107.

Reply:
It should indeed. At the moment this branch is based on existing open PRs. So I've gone ahead and made this change in #23107.